client/app/router.js

```diff
@@ -29,6 +29,7 @@ Router.map(function() { // eslint-disable-line
   this.route('show-geography', { path: '/projects' });
   this.route('disclaimer');
   if (config.showAlerts) { this.route('statuses'); }
+  this.route('subscribed');
```

Comment:
Should put this route behind the feature flag just like `statuses`.

Reply:
Done